apps/web/src/app/[locale]/challenge/[slug]/getChallengeRouteData.ts

```diff
@@ -35,6 +35,11 @@ export const getChallengeRouteData = cache(async (slug: string, session: Session
           userId: session?.user.id || '',
         },
       },
+      submission: {
+        where: {
```

Comment:
we can probably just take 1 thats successful via prisma and void needing to use some below

ex: where isSuccessful is true (i forgot the exact column name)

Reply:
Sure that makes sense. If I make this change, do you want me to move the check for hasSuccessfulSubmission over to the component using it instead of here?

Reply:
@bautistaaa I removed the part where I augment the `challenge` property. So we can just check length in the `Description` component.

Reply:
we need to keep the explicitness of what having a successful solution is

submissions length > 0 meaning will be lost over time.

for now i guess you can do 

```ts
const hasSolved = challenge.submission.length > 0
```

or something in the component below or return it explicitly here